### DISCUSSION TITLE
Feature Request: Save your configuration for use at a later date

### DISCUSSION THREAD

**Discussion by stinkysmellystinkysmelly** [ChrisTitusTech/winutil#448] [Ideas]
I would love to see the ability to save your selections in the UI. If you have seen the software called Ninite, they do this pretty well. Although I doubt this project could use the exact same implementation.

Something like a JSON file generated from the selections in the UI, that you could then import when running the script, would be really nice to see.

**Comment by techguy16** [comment]
That's a great idea.

**Comment by S-Pika** [comment]
Would save a lot of time from having to go through and reselect all the options you want on every windows update or reinstall.

**Comment by DeveloperDurp** [comment]
Once #576 gets merged I will investigate this and attempt to implement if possible.
